### ISSUE TITLE
replace deprecated `get` keyword with `scheduler`

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/pmda#48]
### Expected behaviour
No warnings are raised when using dask.

### Actual behaviour
Using the `get=` kwarg is deprecated, instead use `scheduler=`

```
pmda/test/test_contacts.py::TestContacts::()::test_startframe
  /home/travis/miniconda/envs/test/lib/python3.6/site-packages/dask/base.py:835: UserWarning: The get= keyword has been deprecated. Please use the scheduler= keyword instead with the name of the desired scheduler like 'threads' or 'processes'
    warnings.warn("The get= keyword has been deprecated. "
```

### Code to reproduce the behaviour

see https://travis-ci.org/MDAnalysis/pmda/jobs/397005019

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
(run `python -c "import pmda; print(pmda.__version__)"`)
(run `python -c "import dask; print(dask.__version__)"`)

**Comment by kain88-de** [comment]
I think for a longer period we should support both keywords. This allows people to use also an older dask version and not break scripts with another release.

**Comment by orbeckst** [comment]
Good idea!

--
Oliver Beckstein
email: orbeckst@gmail.com

**Comment by VOD555** [comment]
It seems in the new version of dask (0.20.0) released on 10/26/2018, it raises an error on the use of the get= keyword and set_options (http://docs.dask.org/en/latest/changelog.html).

**Comment by orbeckst** [comment]
> I think for a longer period we should support both keywords. This allows people to use also an older dask version and not break scripts with another release.

Would be nice to support old and new dask, but dask is moving so quickly, I don't think we have the developer time to do this. Instead we will have to require dask ≥ 0.18.0 and have users update. At least that is relatively painless for dask.

**Comment by kain88-de** [comment]
I'm also OK saying this library is cutting edge. We explicitly claim the library is not stable in the readme.
